src/AWS.Deploy.Orchestration/CdkAppSettingsSerializer.cs

```diff
@@ -33,24 +39,19 @@ public string Build(CloudApplication cloudApplication, Recommendation recommenda
                 recommendation.Recipe.Version,
                 session.AWSAccountId,
                 session.AWSRegion,
-                new ()
+                settings: Helpers.GetOptionSettingsContainer(_optionSettingHandler, recommendation, session.ProjectDefinition, _directoryManager, OptionSettingsType.Recipe),
+                deploymentBundleSettings: JsonConvert.SerializeObject(deploymentBundleContainer, Formatting.Indented)
```

Comment:
The `DeloymentBundleSettings` is defined as a `string` property in the `IRecipeProps`. This property only exists so that it can be written to the CloudFormation template metadata and can be retrieved to preload settings during a re-deployment. This property is not used by the CDKTemplates and therefore it makes sense to not store it as any other concrete type apart from `string`